commit d8746acb5fecaed70458dc8135cd8a6c5cd5b498
Author: Gareth Stockwell <gareth.stockwell@accenture.com>
Date:   Mon Oct 10 12:43:13 2011 +0100

    Make output of configure.bat more readable
    
    * Suppress output of failed SBSv2 instances to the console - they are
      available in config.log if required.
    * Remove unnecessary '...'s and 'OK's
    
    Reviewed-by: PENDING

diff --git a/bin/compilercheck.pl b/bin/compilercheck.pl
index 7203a62..825b03f 100644
--- a/bin/compilercheck.pl
+++ b/bin/compilercheck.pl
@@ -46,11 +46,11 @@ my $result = `sbs --query=config[armv5_urel] 2>&1`;
 my $count = length($result);
 my $idx = index($result, "arm.v5.urel.rvct");
 
-print "check sbs release armv5 default compiler...\n";
+print "Checking default compiler for 'sbs -c armv5_urel' ...\n";
 if ($idx > 0 && $count > ($idx + 20)){
 	$version_hig = substr($result, $idx+16, 1);
 	$version_low = substr($result, $idx+18, 1);
-	print "- sbs default release armv5 compiler found: RVCT" . $version_hig . "." . $version_low . "\n";
+	print "Default compiler found: RVCT" . $version_hig . "." . $version_low . "\n";
 
 	# check if RVCT in sbs config is < 4.0
 	if($version_hig < 4 && $version_hig >= 2){
@@ -60,23 +60,23 @@ if ($idx > 0 && $count > ($idx + 20)){
 			$idx = index($result, ".");
 			$rvctversion_hig = substr($result, $idx-1, 1);
 			$rvctversion_low = substr($result, $idx+1, 1);
-			print "- installed RVCT compiler: RVCT". $rvctversion_hig . "." . $rvctversion_low . "\n";
+			print "Installed RVCT compiler: RVCT". $rvctversion_hig . "." . $rvctversion_low . "\n";
 			#check if installed RVCT is >= 4.0
 			if( $rvctversion_hig >= 4){
 				#check if the variable QT_RVCT_VERSION is set to the rvct version
 				$qtvarval = $ENV{'QT_RVCT_VERSION'};
 				$qtvarshould = $rvctversion_hig. "." .$rvctversion_low;
 				if((length($qtvarval) <= 0) || ($qtvarshould != $qtvarval)){
-					print "- Please set enviroment variable QT_RVCT_VERSION=";
-					print "$rvctversion_hig.$rvctversion_low and run configure again!!!\n";
+					print "Error: please set environment variable QT_RVCT_VERSION=";
+					print "$rvctversion_hig.$rvctversion_low and re-run configure\n";
 				}
 			}
 		}
 		else{
-			print "- OK\n";
+			print "OK\n";
 		}
 	}
 }
 else{
-    print "- OK\n";
+    print "OK\n";
 }
diff --git a/configure.bat b/configure.bat
index 0709999..3da25b5 100644
--- a/configure.bat
+++ b/configure.bat
@@ -353,7 +353,8 @@ set MODULE_SERVICEFRAMEWORK_REQUESTED=no
 set MODULE_SYSTEMINFO_REQUESTED=no
 set MODULE_VERSIT_REQUESTED=no
 
-echo Checking selected modules:
+echo.
+echo Checking selected modules ...
 :modulesTag2
 
 for /f "tokens=1,*" %%a in ("%MOBILITY_MODULES_UNPARSED%") do (
@@ -364,50 +365,50 @@ for /f "tokens=1,*" %%a in ("%MOBILITY_MODULES_UNPARSED%") do (
 : What we want is a switch as we need to check module name and only want to
 : distinguish between false and correct module names being passed
 if %FIRST% == bearer (
-    echo     Bearer Management selected
+    echo Bearer management
     set MODULE_BEARER_REQUESTED=yes
 ) else if %FIRST% == contacts (
-    echo     Contacts selected
+    echo Contacts
     set MODULE_CONTACTS_REQUESTED=yes
 ) else if %FIRST% == location (
-    echo     Location selected
+    echo Location
     set MODULE_LOCATION_REQUESTED=yes
 ) else if %FIRST% == messaging (
-    echo     Messaging selected
+    echo Messaging
     set MODULE_MESSAGING_REQUESTED=yes
 ) else if %FIRST% == multimedia (
-    echo     Multimedia selected
+    echo Multimedia
     set MODULE_MULTIMEDIA_REQUESTED=yes
 ) else if %FIRST% == publishsubscribe (
-    echo     PublishSubscribe selected
+    echo Publish and subscribe
     set MODULE_PUBLISHSUBSCRIBE_REQUESTED=yes
 ) else if %FIRST% == systeminfo (
-    echo     Systeminfo selected
+    echo Systeminfo
     set MODULE_SYSTEMINFO_REQUESTED=yes
 ) else if %FIRST% == serviceframework (
-    echo     ServiceFramework selected
+    echo Service framework
     set MODULE_SERVICEFRAMEWORK_REQUESTED=yes
 ) else if %FIRST% == versit (
-    echo     Versit selected ^(implies Contacts^)
+    echo Versit ^(implies Contacts^)
     set MODULE_CONTACTS_REQUESTED=yes
     set MODULE_VERSIT_REQUESTED=yes
 ) else if %FIRST% == organizer (
-    echo     Organizer selected
+    echo Organizer
     set MODULE_ORGANIZER_REQUESTED=yes
 ) else if %FIRST% == feedback (
-    echo     Feedback selected
+    echo Feedback
     set MODULE_FEEDBACK_REQUESTED=yes
 ) else if %FIRST% == sensors (
-    echo     Sensors selected
+    echo Sensors
     set MODULE_SENSORS_REQUESTED=yes
 ) else if %FIRST% == gallery (
-    echo     Gallery selected
+    echo Gallery
     set MODULE_GALLERY_REQUESTED=yes
 ) else if %FIRST% == connectivity (
-    echo     Connectivity selected
+    echo Connectivity
     set MODULE_CONNECTIVITY_REQUESTED=yes
 ) else (
-    echo     Unknown module %FIRST%
+    echo Error: unknown module %FIRST%
     goto errorTag
 )
 
@@ -432,7 +433,6 @@ set RELEASEMODE=
 set WIN32_RELEASEMODE=
 
 set CURRENTDIR=%CD%
-echo %CURRENTDIR%
 if exist %QT_MOBILITY_PREFIX% goto prefixExists
 mkdir %QT_MOBILITY_PREFIX%
 if errorlevel 1 goto invalidPrefix
@@ -487,7 +487,8 @@ echo mobility_modules = %MOBILITY_MODULES%  >> %PROJECT_CONFIG%
 echo contains(mobility_modules,versit): mobility_modules *= contacts >> %PROJECT_CONFIG%
 echo contains(mobility_modules,connectivity): mobility_modules *= serviceframework >> %PROJECT_CONFIG%
 
-echo Checking available Qt
+echo.
+echo Checking available Qt ...
 call %QT_PATH%qmake -v >> %PROJECT_LOG% 2>&1
 if errorlevel 1 goto qmakeNotFound
 goto qmakeFound
@@ -529,35 +530,36 @@ setlocal
     if "%BUILDSYSTEM%" == "symbian-abld" (
         call make -h >> %PROJECT_LOG% 2>&1
         if not errorlevel 1 (
-            echo ... Symbian abld make found.
+            echo Symbian abld
             set MAKE=make
         )
     ) else if "%BUILDSYSTEM%" == "symbian-sbsv2" (
         call make -h >> %PROJECT_LOG% 2>&1
         if not errorlevel 1 (
-            echo ... Symbian sbsv2 make found.
+            echo Symbian sbsv2
             set MAKE=make
         )
     ) else if "%BUILDSYSTEM%" == "win32-nmake" (
         call nmake /? >> %PROJECT_LOG% 2>&1
         if not errorlevel 1 (
-            echo ... nmake found.
+            echo nmake
             set MAKE=nmake
         )
     ) else if "%BUILDSYSTEM%" == "win32-mingw" (
         call mingw32-make -v >> %PROJECT_LOG% 2>&1
         if not errorlevel 1 (
-            echo ... mingw32-make found.
+            echo mingw32-make
             set MAKE=mingw32-make
         )
     ) else (
-        echo ... Unknown target environment %BUILDSYSTEM%.
+        echo Error: unknown target environment %BUILDSYSTEM%.
     )
     call cd %CURRENT_PWD%
 call endlocal&set %1=%MAKE%&set %2=%BUILDSYSTEM%&goto :EOF
 
 :checkMake
-echo Checking make
+echo.
+echo Checking make ...
 call :makeTest MOBILITY_MAKE MOBILITY_BUILDSYSTEM
 if not "%MOBILITY_MAKE%" == "" goto compileTests
 
@@ -585,7 +587,7 @@ setlocal
     set FAILED=0
     if "%MOBILITY_BUILDSYSTEM%" == "symbian-sbsv2" (
         call %MOBILITY_MAKE% release-armv5 >> %PROJECT_LOG% 2>&1
-        for /f "tokens=2" %%i in ('%MOBILITY_MAKE% release-armv5 SBS^="@sbs --check"') do set FAILED=1
+        for /f "tokens=2" %%i in ('%MOBILITY_MAKE% release-armv5 SBS^="@sbs --check" 2^>^&1') do set FAILED=1
     ) else if "%MOBILITY_BUILDSYSTEM%" == "symbian-abld" (
         call %MOBILITY_MAKE% release-gcce >> %PROJECT_LOG% 2>&1
         for /f "tokens=2" %%i in ('%MOBILITY_MAKE% release-gcce ABLD^="@ABLD.BAT -c" 2^>^&1') do if not %%i == bldfiles set FAILED=1
@@ -621,28 +623,37 @@ REM compile tests go here.
 for /f "tokens=3" %%i in ('call %QT_PATH%qmake %SOURCE_PATH%\config.tests\make\make.pro 2^>^&1 1^>NUL') do set BUILDSYSTEM=%%i
 if "%BUILDSYSTEM%" == "symbian-abld" goto symbianTests
 if "%BUILDSYSTEM%" == "symbian-sbsv2" (
-  perl -S  %SOURCE_PATH%\bin\compilercheck.pl
-  goto symbianTests
+    echo.
+    perl -S  %SOURCE_PATH%\bin\compilercheck.pl
+    goto symbianTests
 )
 goto windowsTests
 
 :symbianTests
 
 if "%MODULE_BEARER_REQUESTED%" == "yes" (
+    echo.
+    echo Running compile tests for bearer module ...
     call :compileTest OCC occ
     call :compileTest SNAP snap
 )
 
 if "%MODULE_CONTACTS_REQUESTED%" == "yes" IF "%MODULE_SYSTEMINFO_REQUESTED%" == "yes" (
+    echo.
+    echo Running compile tests for contacts / systeminfo module ...
     call :compileTest SymbianContactSIM symbiancntsim
 )
 
 if "%MODULE_CONTACTS_REQUESTED%" == "yes" (
+    echo.
+    echo Running compile tests for contacts module ...
     call :compileTest SymbianContactModel symbiancntmodel
     call :compileTest SymbianContactModelv2 symbiancntmodelv2
 )
 
 if "%MODULE_CONNECTIVITY_REQUESTED%" == "yes" (
+    echo.
+    echo Running compile tests for connectivity module ...
     call :compileTest BTEngineConnectionManager_Symbian btengconnman_symbian
     call :compileTest BTEngineDeviceManager_Symbian btengdevman_symbian
     if "%NFC_SYMBIAN%" == "auto" (
@@ -653,28 +664,38 @@ if "%MODULE_CONNECTIVITY_REQUESTED%" == "yes" (
 )
 
 if "%MODULE_FEEDBACK_REQUESTED%" == "yes" (
+    echo.
+    echo Running compile tests for feedback module ...
     call :compileTest IMMERSION immersion
     call :compileTest AdvancedTouchFeedback advancedtouchfeedback
     call :compileTest CHWRMHaptics chwrmhaptics
 )
 
 if "%MODULE_GALLERY_REQUESTED%" == "yes" (
+    echo.
+    echo Running compile tests for gallery module ...
     call :compileTest MDS mds
     call :compileTest MDS_25 mds_25
     call :compileTest MDS_25_92MCL mds_25_92mcl
 )
 
 if "%MODULE_LOCATION_REQUESTED%" == "yes" (
+    echo.
+    echo Running compile tests for location module ...
     call :compileTest LBT lbt
     call :compileTest location_fix location_fix
 )
 
 if "%MODULE_MESSAGING_REQUESTED%" == "yes" (
+    echo.
+    echo Running compile tests for messaging module ...
     call :compileTest Symbian_Messaging_Freestyle messaging_freestyle
     call :compileTest Symbian_Messaging_Freestyle_MAPI12 messaging_freestyle_mapi12
 )
 
 if "%MODULE_MULTIMEDIA_REQUESTED%" == "yes" (
+    echo.
+    echo Running compile tests for multimedia module ...
     call :compileTest Audiorouting_s60 audiorouting_s60
     call :compileTest mmf_http_cookies mmf_http_cookies
     REM Currently not being supported
@@ -689,6 +710,8 @@ if "%MODULE_MULTIMEDIA_REQUESTED%" == "yes" (
 )
 
 if "%MODULE_ORGANIZER_REQUESTED%" == "yes" (
+    echo.
+    echo Running compile tests for organizer module ...
     call :compileTest SymbianENote symbianenote
 )
 
@@ -697,6 +720,8 @@ if "%MODULE_PUBLISHSUBSCRIBE_REQUESTED%" == "yes" (
 )
 
 if "%MODULE_SENSORS_REQUESTED%" == "yes" (
+    echo.
+    echo Running compile tests for sensors module ...
     call :compileTest S60_Sensor_API sensors_s60_31
     call :compileTest Symbian_Sensor_Framework sensors_symbian
     call :compileTest Sensor_Framework_Light sensors_symbian_light
@@ -707,6 +732,8 @@ if "%MODULE_SERVICEFRAMEWORK_REQUESTED%" == "yes" (
 )
 
 if "%MODULE_SYSTEMINFO_REQUESTED%" == "yes" (
+    echo.
+    echo Running compile tests for systeminfo module ...
     call :compileTest Symbian_Hb hb_symbian
     call :compileTest FmTxClientCheck FmTxClient
     call :compileTest LockandFlipKeys LockandFlipPSkeys
@@ -733,8 +760,8 @@ call :compileTest WindowsMediaFoundation wmf
 
 :noTests
 
-echo End of compile tests
 echo.
+echo End of compile tests
 echo.
 goto processHeaders